docker/docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
     ports:
       - "5432:5432"
     volumes:
-      - "./volumes/database:/var/lib/postgresql"
+      - "$HOME/docker/volumes/postgres:/var/lib/postgresql"
```

Comment:
How about something like `"${POSTGRES_VOLUME_DIR:-$HOME/docker/volumes/postgresql}:/var/lib/postgresql"
` so it's possible for someone to use a different directory if they want?

Reply:
Good call